index.html

```diff
@@ -62,8 +62,203 @@ <h1>I’m Taflon Glad to see you!</h1>
+                </div>
+
+               
+               
+            </div>
+          
+        </section>
+        <section class="about_page">
+            <div class="main_container">
+               <h2 class="about_myself_header"> About Myself</h2> 
+               <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+
+               </p>
+               <div class="connect">
+                LET’S CONNECT
+               </div>
+               <ul>
+                 <li>
+                    <a href="#">
+                        <img src="images/Vector-twiter.png" alt="">
+                    </a>
+                 </li>
+                 <li><a href="#">
+                     <img src="images/Linkedin icon.png" alt="">
+                   </a>
+                 </li>
+                 <li>
+                    <a href="#">
+                        <img src="images/Vector_M.png" alt="">
+                    </a>
+                 </li>
+                 <li>
+                    <a href="#">
+                        <img src="images/Vector_peace.png" alt="">
+                    </a>
+                 </li>
+                 <li>
+                    <a href="#"><img src="images/Vector_dog.png" alt="">
+                    </a>
+                 </li>
+               </ul>
+               <div class="about-button">
+                <button>Get my resume</button>
+             </div>
+            </div>
+
+            <div class="languages_container">
+                <div class="languages_wrapper">
+                    <div class="icon_one">
+                        <h4>Languages</h4>
+                        <i class="fa-solid fa-chevron-right"></i>
+                    </div>
+                    <div class="languages">
+                        <div class="languages_details">
+                            <img src="images/js.png" alt="">
+                          <p>Javascript</p> 
+                        </div>
+                        <div class="languages_details">
+                            <img src="images/html.png" alt="">
+                           <p>Html</p>
+                        </div>
+                        <div class="languages_details">
+                            <img src="images/css.png" alt="">
+                            <p>css</p>
+                        </div>
+                    </div>
+                   
+                    <div class="icon_one">
+                        <h4>Frameworks</h4>
+                        <i class="fa-solid fa-chevron-right"></i>
+                    </div>
+                    <hr>
+                    <div class="icon_one">
+                        <h4>Skills</h4>
+                        <i class="fa-solid fa-chevron-right"></i>
+                    </div>
+                    <hr>
```

Comment:
- [ ] Dear @Tafloninno, Please remember its among the primary requirement to make the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`. 
- [ ] Also, The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li`